src/Tribe/Views/V2/Hooks.php

```diff
@@ -713,4 +715,28 @@ public function print_single_json_ld() {
 
 		$this->container->make( Template\JSON_LD::class )->print_single_json_ld();
 	}
+
+	/**
```

Comment:
:no_entry_sign: Doc comment for parameter "$css_template" missing

<sub>hash: 63da2489f20fe951a0b5f8ae4a07d85d</sub>